Production/crab/Run3Summer23BPixDRPremix_HLT/mc.yaml

```diff
@@ -0,0 +1,29 @@
+config:
+  params:
+    cmsRunCfg: hlt_test.py
+
+
+
+DYTo2L_MLL-4to50: /DYTo2L_MLL-4to50_TuneCP5_13p6TeV_pythia8/Run3Summer23BPixDRPremix-130X_mcRun3_2023_realistic_postBPix_v2-v1/GEN-SIM-RAW 
```

Comment:
Do we want both Run3Summer23BPix and Run3Winter24?

Reply:
I have added samples from campaign "Run3Summer23BPixDRPremix" after @brallmond requested.
@brallmond, can you please answer. Thanks!